packages/dev/core/src/PostProcesses/RenderPipeline/Pipelines/ssaoRenderingPipeline.ts

```diff
@@ -340,29 +341,17 @@ export class SSAORenderingPipeline extends PostProcessRenderPipeline {
     private _createRandomTexture(): void {
         const size = 512;
 
-        this._randomTexture = new DynamicTexture("SSAORandomTexture", size, this._scene, false, Texture.TRILINEAR_SAMPLINGMODE);
-        this._randomTexture.wrapU = Texture.WRAP_ADDRESSMODE;
-        this._randomTexture.wrapV = Texture.WRAP_ADDRESSMODE;
-
-        const context = this._randomTexture.getContext();
-
-        const rand = (min: number, max: number) => {
-            return Math.random() * (max - min) + min;
-        };
-
-        const randVector = Vector3.Zero();
-
-        for (let x = 0; x < size; x++) {
-            for (let y = 0; y < size; y++) {
-                randVector.x = Math.floor(Math.max(0.0, rand(-1.0, 1.0)) * 255);
-                randVector.y = Math.floor(Math.max(0.0, rand(-1.0, 1.0)) * 255);
-                randVector.z = Math.floor(Math.max(0.0, rand(-1.0, 1.0)) * 255);
-
-                context.fillStyle = "rgb(" + randVector.x + ", " + randVector.y + ", " + randVector.z + ")";
-                context.fillRect(x, y, 1, 1);
-            }
+        const data = new Uint8Array(size * size * 3);
+        for (let index = 0; index < data.length; ) {
+            data[index++] = Math.floor(Math.max(0.0, Scalar.RandomRange(-1.0, 1.0)) * 255);
+            data[index++] = Math.floor(Math.max(0.0, Scalar.RandomRange(-1.0, 1.0)) * 255);
+            data[index++] = Math.floor(Math.max(0.0, Scalar.RandomRange(-1.0, 1.0)) * 255);
         }
 
-        this._randomTexture.update(false);
+        const texture = RawTexture.CreateRGBTexture(data, size, size, this._scene, false, false, Constants.TEXTURE_BILINEAR_SAMPLINGMODE);

```

Comment:
@sebavan I just noticed I inadvertently changed this from trilinear to bilinear. Should I change it back?

Reply:
No, it does not make any difference, TRI will fallback to BI if there is no mipmaps. In any case, BI is better as we do know there is no mips.